src/main/java/net/masterthought/cucumber/json/deserializers/StatusDeserializer.java

```diff
@@ -21,6 +22,7 @@
     @Override
     public Status deserialize(JsonNode rootNode, Configuration configuration) {
 
+        Locale.setDefault(Locale.US);
```

Comment:
Best would be to restore previous locale once below code is completed

Reply:
hi @damianszczepanik 
After spending some time on this, I believe the best way to restore previous locale is to pass Locale.ENGLISH parameter in the firstplace for each method that are used for status strings that are intended to be interpreted locale independently, such as PASSED, FAILED etc (These are English words anyway) rather than changing default locale.

No args toUpperCase() returns toUpperCase(Locale.getDefault()) so there is no difference between calling "toUpperCase() which in turn calls toUpperCase(Locale.getDefault())" and "toUpperCase(Locale.ENGLISH)"

Let me know what your opinion is and I can make necessary updates. I already identified two files that would require changes.
This is the same approach as my first commit for this pull request but updating only bare minimum files this time.